net/udp/udp_sendto_buffered.c

```diff
@@ -394,10 +389,9 @@ static uint16_t sendto_eventhandler(FAR struct net_driver_s *dev,
                                     FAR void *pvconn, FAR void *pvpriv,
                                     uint16_t flags)
 {
-  FAR struct udp_conn_s *conn = (FAR struct udp_conn_s *)pvconn;
-  FAR struct socket *psock = (FAR struct socket *)pvpriv;
+  FAR struct udp_conn_s *conn = (FAR struct udp_conn_s *)pvpriv;
```

Comment:
Let's document here:

 ```
/* FAR struct udp_conn_s *conn = (FAR struct udp_conn_s *)pvconn;
  *
  * Do not use pvconn argument to get the UDP connection pointer (the above
  * commented line) because pvconn is normally NULL for some events like
  * NETDEV_DOWN. Instead, the UDP connection pointer can be reliably
  * obtained from the corresponding UDP socket.
  */
```

Reply:
Thanks for reminding, There are too many places to use pvconn, I do not want to add the same comment everywhere, I will plan to remove the pvconn parameter on next patch,